### PR TITLE
[BACKLOG-31898] CSRF - /ServiceAction

### PR DESCRIPTION
**Pull request by davidmsantos90**
@pentaho/millenniumfalcon please review

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 39s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl assemblies/data-access-plugin
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: assemblies/data-access-plugin/src/main/resources/resources/web/pentaho-ajax.js [hunk 1]
```diff
@@ -12,327 +12,304 @@
...
 var STATUS_OK = 200;
 var STATUS_UNAUTHORIZED = 401;
 var STATUS_NOT_FOUND = 404;
+
 var base = '';
 
- document.write("<!-- webcontext.js is written by PentahoWebContextFilter. Content of this file contains values of CONTEXT_PATH and FULLY_QUALIFIED_SERVER_URL --> \n"); 
- document.write("<script type='text/javascript' src='webcontext.js'></script>"); 
+document.write("<!-- webcontext.js is written by PentahoWebContextFilter. Content of this file contains values of CONTEXT_PATH and FULLY_QUALIFIED_SERVER_URL --> \n");
+document.write("<script type='text/javascript' src='webcontext.js'></script>");
```
Comment (dcleao) [inline assemblies/data-access-plugin/src/main/resources/resources/web/pentaho-ajax.js]: This seems to be the only relevant change of this file, in relation to the common-ui's version.
We can probably still load the central version and only do this here.

File: assemblies/data-access-plugin/src/main/resources/resources/test/query-sample.html [hunk 2]
```diff
@@ -13,6 +13,7 @@
 </script>
 <script language="javascript" src="../web/dojo/dojo.js" type="text/javascript"></script>
 <script language="javascript" src="../web/messages/Messages.js" type="text/javascript"></script>
+<script language="javascript" src="../web/pentaho-csrf.js" type="text/javascript"></script>
 <script language="javascript" src="../web/pentaho-ajax.js" type="text/javascript"></script>
```
Comment (dcleao) [inline assemblies/data-access-plugin/src/main/resources/resources/test/query-sample.html]: Only need the `pentaho-ajax` include, if it then loads common-ui's pentaho-ajax.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 42s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl assemblies/data-access-plugin
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 37s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl assemblies/data-access-plugin
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by davidmsantos90** [comment]
@pentaho/millenniumfalcon pr updated. please review

**Pull request merged**